src/CodeStyle/CSharp/CodeFixes/CSharpCodeStyleFixes.csproj

```diff
@@ -48,14 +46,14 @@
     <InternalsVisibleToTest Include="Microsoft.CodeAnalysis.CSharp.CodeStyle.Tests" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="CSharpCodeStyleFixesResources.Designer.cs">
+    <Compile Update="CSharpCodeStyleFixesResources.Designer.cs">
```

Comment:
This is new MSBuild syntax for adding metadata to an existing item. Since the default glob would have picked up this .cs file there will be a Compile item for this file already. So another Compile Include would mention this file twice. Instead, the Update syntax adds the metadata that's needed, by the resx designer in this case, to an existing msbuild item.